front/lib/labs/config.ts

```diff
@@ -0,0 +1,24 @@
+import { EnvironmentConfig } from "@dust-tt/types";
+
+const config = {
+  getNangoPublicKey: (): string => {
+    return EnvironmentConfig.getEnvVariable("NANGO_PUBLIC_KEY");
+  },
+  getNangoSecretKey: (): string => {
+    return EnvironmentConfig.getEnvVariable("NANGO_SECRET_KEY");
+  },
+  getNangoGoogleDriveConnectorId: (): string => {
+    return EnvironmentConfig.getEnvVariable("NANGO_GOOGLE_DRIVE_CONNECTOR_ID");
+  },
+  getLabsApiKey: (): string => {
+    return EnvironmentConfig.getEnvVariable("LABS_API_KEY");
+  },
+  getLabsWorkspaceId: (): string => {
```

Comment:
What do we store in this variable? Shouldn't the dust api be called on the user's workspace?